samples/dnn/js_face_recognition.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <script src="opencv.js" type="text/javascript"></script>
```

Comment:
You may consider to use async loading of opencv.js because it is big. Utils provides a tool to async load it. See https://github.com/opencv/opencv/blob/master/doc/js_tutorials/js_assets/utils.js#L6

Reply:
@huningxin, Am I right that `onload` callback of `<body>` executes when all scripts have been loaded? Despite they are sync or async.

Reply:
To my understanding, you are right.